router/batchrouter/batchrouter.go

```diff
@@ -2233,6 +2256,8 @@ func (brt *HandleT) Setup(backendConfig backendconfig.BackendConfig, jobsDB, err
 	config.RegisterDurationConfigVariable(180, &brt.retryTimeWindow, true, time.Minute, []string{"BatchRouter." + brt.destType + "." + "retryTimeWindow", "BatchRouter." + brt.destType + "." + "retryTimeWindowInMins", "BatchRouter." + "retryTimeWindow", "BatchRouter." + "retryTimeWindowInMins"}...)
 	config.RegisterDurationConfigVariable(30, &brt.asyncUploadTimeout, true, time.Minute, []string{"BatchRouter." + brt.destType + "." + "asyncUploadTimeout", "BatchRouter." + "asyncUploadTimeout"}...)
 	config.RegisterInt64ConfigVariable(1*bytesize.GB, &brt.payloadLimit, true, 1, []string{"BatchRouter." + brt.destType + "." + "PayloadLimit", "BatchRouter.PayloadLimit"}...)
+	config.RegisterDurationConfigVariable(90, &brt.jobdDBRequestTimeout, true, time.Second, []string{"JobsDB." + "BatchRouter." + "CommandRequestTimeout", "JobsDB." + "CommandRequestTimeout"}...)
```

Comment:
Increase defaults to a little bigger number?

Reply:
We chose the numbers based on hosted, where the maximum duration for the past 30 days is `30sec`.
Do we have a worse performing environment than hosted?

Reply:
I have seen worse times than 30 sec in the past. Not lately though. 

I see the value is 90 sec now. Im cool with that.